api/exec/compose_stack_test.go

```diff
@@ -0,0 +1,66 @@
+package exec
```

Comment:
could it be that base branch is not rebased with develop, while this is? It seems like develop has this already

Reply:
I'm afraid it hasn't been updated to use the wrapper lib. It sounds like we may be looking at the different sources, here's what we have it develop: `https://github.com/portainer/portainer/blob/develop/api/exec/compose_wrapper.go` and it doesn't use docker-compose-wrapper

Reply:
ah, yes. https://portainer.atlassian.net/browse/EE-1069 is still in testing